Makefile

```diff
@@ -192,7 +192,10 @@ gogen: ensure-gopath
 	go generate ./util/argo/...
 
 .PHONY: protogen
-protogen: ensure-gopath mod-vendor-local
+protogen: mod-vendor-local protogen-fast
```

Comment:
Great stuff! Would it be worth adding `install-go-tools-local` to the task as well? It's not so clear that this is required for this to work.

Reply:
The doc specifies to setup the local toolchain with `make install-tools-local` which will do the installation. I think this is enough and I do not really like having heavy steps always executed on make.